newIDE/app/src/MainFrame/ElectronMainMenu.js

```diff
@@ -193,7 +193,7 @@ const buildAndSendMenuTemplate = (
         enabled: !!project,
       },
       {
-        label: i18n._(t`Show Start Page`),
+        label: i18n._(t`Show Home`),
         onClickSendEvent: 'main-menu-open-start-page',
```

Comment:
@4ian I suppose we shouldn't change the event id for tracking purpose?

Reply:
Let's keep the same yes!